### DISCUSSION TITLE
Windows Size

### DISCUSSION THREAD

**Discussion by B16f00t** [Akascape/CTkMessagebox#11] [Ideas]
When the message to be displayed is too large the window size is not adjusted.
I think there should be a default size set and the size should be adjusted if the text is large like messagebox does in tkinter.

And another thing the question icon that comes by default when working in light mode, half of it blurs with the background.

By the way I love your customtkinter improvements.

**Comment by Akascape** [comment]
@B16f00t 
> I think there should be a default size set and the size should be adjusted if the text is large like messagebox does in tkinter.

I have implemented this, now the window height will be resized automatically in this case.

> And another thing the question icon that comes by default when working in light mode, half of it blurs with the background

I have also changed the contrast a bit in the question png icon, please download the latest version. :)
